packit/status.py

```diff
@@ -150,8 +150,23 @@ def get_updates(self, number_of_updates: int = 3) -> List:
             "updates"
         ]
         logger.debug("Bodhi updates fetched.")
-        results = results[:number_of_updates]
 
-        return [
-            [result["title"], result["karma"], result["status"]] for result in results
+        stable_branches: Set[str] = set()
+        all_updates = [
+            [
+                result["title"],
+                result["karma"],
+                result["status"],
+                result["release"]["branch"],
+            ]
+            for result in results
         ]
+        updates = []
+        for [update, karma, status, branch] in all_updates:
+            if branch not in stable_branches or status != "stable":
```

Comment:
I know we don't comment our code overly, but your code can stand out in a crowd :-)

```suggestion
            # Don't return more than one stable update per branch
            if branch not in stable_branches or status != "stable":
```